templates/land-see/kim-myeong.amp.html

```diff
@@ -55,15 +55,147 @@
+          {{{post-copy}}}
+        </p>
+      </div>
+      {{/land-see-post-detail}}
+
+      {{#land-see-copy}}
+        {{#subheader}}
+        <h2 class="land-see-story-subheader max-width-3 mx-auto pb0 pt4 px4">{{.}}</h2>
+        {{/subheader}}
+        {{#paragraph}}
+        <p class="land-see-story-copy max-width-3 mx-auto py2 px4">
+          {{{.}}}
+        </p>
+        {{/paragraph}}
+        {{#blockquote}}
+        <blockquote class="land-see-blockquote max-width-3 mx-auto py4 px4 {{classes}} "{{#cite-url}}cite="{{.}}"{{/cite-url}}>
+          <p>{{quote}}</p>
+          {{#citation}}
+          <cite>{{.}}</cite>
+          {{/citation}}
+        </blockquote>
+        {{/blockquote}}
+      {{/land-see-copy}}
+
+      {{#land-see-image-gallery}}
+      <div class="land-see-image-gallery mx-auto">
+        <div class="clearfix mx1">
+          {{#carousel}}
+          <amp-carousel
+            id="{{id}}"
+            class="{{classes}} mx-auto"
+            {{#media-query}}
+            media="{{media-query}}"
+            {{/media-query}}
+            width="{{width}}"
+            height="{{height}}"
+            layout="{{layout}}"
+            [slide]="selectedSlide"
+            on="slideChange:AMP.setState({selectedSlide: event.index})"
+            controls
+            loop
+            type="slides">
+              {{#image-slide}}
+              <amp-img
+                class="{{classes}}"
+                alt="{{alt-text}}"
+                width="{{width}}"
+                height="{{height}}"
+                layout="{{layout}}"
+                src="{{src}}">
+              </amp-img>
+              {{/image-slide}}
+          </amp-carousel>
+          {{#story-preview}}
+          <div class="land-see-gallery-progress mt1 center col-12 overflow-hidden">
+            <div class="land-see-gallery-progress-scroll col-12 nowrap">
+              {{#image-slide}}
+              <button on="{{on-action}}" class="{{button-classes-preview}}">
+                <amp-img
+                  class="{{classes}} {{initial-classes}}"
+                  [class]="{{dynamic-classes}}"
+                  alt="{{alt-text-preview}}"
+                  width="77"
+                  height="58"
+                  layout="fixed"
+                  src="{{src}}">
+                </amp-img>
+              </button>
+              {{/image-slide}}
+            </div>
+          </div>
+          {{/story-preview}}
+          {{/carousel}}
+        </div>
+      </div>
+      {{/land-see-image-gallery}}
+
+      {{#land-see-copy-suffix}}
+      <p class="land-see-story-copy max-width-3 mx-auto pt1 pb3 px4">
+        {{{.}}}
       </p>
-    </div>
+      {{/land-see-copy-suffix}}
+    </section>
+    {{/land-see-story-content}}
+
+    {{#land-see-instagram}}
+    <section class="land-see-story-instagram-container my4 mx-auto center">
+      <section class="{{classes}}">
```

Comment:
do you need this wrapping section, it feels weird to wrap just one section inside a section, was there some semantics that you had in mind for this ?

Reply:
I'm trying to reuse the code from the home template. That called for an accordion where this implementation does not. The nested sections was a consequence of removing the `amp-accordion` tag without changing the underlying markup. Reviewing the CSS, however, I see that we can get away with using a div instead of a nested section.

Reply:
can you point me to what you were trying to reuse, may be there is an easier way

Reply:
Oh, by "reuse" I meant use the same styles as the `amp-accordion` story categories on the main page. The instagram section is copied in part from `partials/story-categories.snip.html`, but it features the instagram svg in the title and doesn't contract or expand.

Reply:
if you can move the classes one parent up will it affect anything?

Reply:
Just a slight style change and that worked a treat! Thanks for the help, I'll commit here in a bit.